go.mod

```diff
@@ -21,29 +21,29 @@ require (
 	github.com/ipfs/go-metrics-prometheus v0.0.2
 	github.com/ipfs/go-unixfsnode v1.9.0
 	github.com/ipld/go-codec-dagpb v1.6.0
-	github.com/libp2p/go-libp2p v0.33.2
+	github.com/libp2p/go-libp2p v0.34.1
 	github.com/libp2p/go-libp2p-kad-dht v0.25.2
 	github.com/libp2p/go-libp2p-record v0.2.0
-	github.com/libp2p/go-libp2p-routing-helpers v0.7.3
+	github.com/libp2p/go-libp2p-routing-helpers v0.7.4-0.20240522201531-0f0528c5efc6
```

Comment:
TODO: update

Reply:
Is based on https://github.com/libp2p/go-libp2p-routing-helpers/pull/84

Reply:
- [ ] @lidel  will switch to v0.7.4 once https://github.com/libp2p/go-libp2p-routing-helpers/pull/85 is done.